docs/source/remote_data.rst

```diff
@@ -542,6 +543,42 @@ Download currency historical rate from `Oanda <https://www.oanda.com/>`__.
 
       [153 rows x 3 columns]
 
+.. _remote_data.oandarest_curr_hist
+
+Oanda REST currency historical rate
+==============================
+
+Download currency historical rate from `Oanda <https://www.oanda.com/>`__.
+
+.. code-block:: python
+
+    In [1]: import pandas_datareader.data as web
+    In [2]: start, end = "2016-01-01", "2016-06-01"
```

Comment:
It may be a good idea to put a shorter duration (to avoid too much request by people who just run doc example)